documentation/src/main/asciidoc/topics/config_examples/default_cache_container.xml

```diff
@@ -0,0 +1,9 @@
+<!-- Creates a Cache Manager named "default" and exports metrics. -->
+<cache-container name="default"
+                 statistics="true">
+  <!-- Adds cluster transport that uses the default JGroups TCP stack. -->
+  <!-- Node names must be unique across the cluster. The default uses a unified hostname. -->
```

Comment:
I think we can remove this comment. Non-unique node names are confusing, but they're not a problem. And I'm not sure what we mean by "a unified hostname"

Reply:
Thanks, @danberindei. TBH I'm not sure about "unified hostname" either. I actually meant to include a comment in the review for Tristan about that but it was late at night and I forgot.